guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -31,8 +31,8 @@ Selected organizations are assigned to users created from the LDAP authenticatio
 . Click *Submit*.
 . Configure new accounts for LDAP users:
 +
-* If you did not select *Automatically Create Accounts In {Project}* checkbox, see xref:Creating_a_User_{context}[] to create user accounts manually.
+* If you did not select *Automatically Create Accounts In {Project}* checkbox, see {AdministeringDocURL}Creating_a_User_{project-context}[Creating a User] in _{AdministeringDocTitle}_ to create user accounts manually.
 +
 * If you selected the *Automatically Create Accounts In {Project}* checkbox, LDAP users can now log in to {Project} using their LDAP accounts and passwords.
 After they log in for the first time, the {Project} administrator has to assign roles to them manually.
-See xref:Assigning_Roles_to_a_User_{context}[] to assign user accounts appropriate roles in {Project}.
+See {AdministeringDocURL}Assigning_Roles_to_a_User_{project-context}[Assigning Roles to a User] in _{AdministeringDocTitle}_ to assign user accounts appropriate roles in {Project}.
```

Comment:
```suggestion
For more information on assigning user accounts appropriate roles in {Project}, see {AdministeringDocURL}Assigning_Roles_to_a_User_admin[Assigning Roles to a User] in _{AdministeringDocTitle}_.
```

Reply:
Same comment as above:
Well I am confused. In your comment above, posted 3 weeks ago, you said the following: "I think the links should use {project-context} ...". 
For this reason, I replaced all "admin" occurrences with {project-context}. 
Now you seem to be saying that I should do the reverse operation, i.e. replace {project-context} with "admin".
Can you please explain if there is something I am getting wrong?